var/spack/repos/builtin/packages/tpl-builder/package.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class TplBuilder(CMakePackage, CudaPackage, ROCmPackage):
+    homepage = "https://github.com/AdvancedMultiPhysics/TPL-builder"
+    git = "https://github.com/AdvancedMultiPhysics/TPL-builder.git"
+
+    maintainers("bobby-philip", "gllongo", "rbberger")
+
```

Comment:
Would you be able to add the license directive between maintainers and versions?